### PR TITLE
Add validation check to the MASTER_URL value.

### PR DESCRIPTION
**Pull request by mwringe**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1351730

### CODE REVIEW

File: deployer/scripts/preflight.sh [hunk 1]
```diff
@@ -27,6 +27,18 @@ function validate_master_accessible() {
   return 1
 }
 
+function validate_master_url() {
+  #Quick check to make sure that the master_url does not contain a '/' at the end
+  #See https://bugzilla.redhat.com/show_bug.cgi?id=1351730
+  if [[ "${master_url: -1}" == "/" ]]; then
```
Comment (jcantrill) [inline deployer/scripts/preflight.sh]: couldnt we just trim it?

**Comment by jcantrill** [comment]
LGTM other then the one nit

**Comment by mwringe** [comment]
Updated. We no longer have a preflight check but we just directly remove the / if present.

**Comment by mwringe** [comment]
[merge]

**Comment by sosiouxme** [comment]
LGTM now

**Pull request merged**